Src/UIComponents/Dashboard/Dashboard.ts

```diff
@@ -21,7 +21,7 @@ function dashboard(components: {'overview': HTMLElement,
     const myDiaryExercises = createElement('li', ['submenu-option'], `Exercises`, 'diary-exercises');
     const myGoals = createElement('div', ['menu-option'], `${myGoalsSVG} My goals`, 'goals');
     const myWeights = createElement('div', ['menu-option'], `${myWeightsSVG} My weights`, 'weights');
-    const profileBtn = createElement('button', ['profile-btn'], `${userSVG} Username ${arrowSVG}`, 'profile');
+    const profileBtn = createElement('button', ['profile-btn'], `${userSVG} ${username} ${arrowSVG}`, 'profile');
     [overview, myGoals, myDiary, myDiaryFood, myDiaryExercises, myWeights, profileBtn].forEach((element) => {
```

Comment:
maybe it would be better to assign that array to variable, bcs you use it again in the line 34, so if you'd like to add one more element, you can do it just modifying things in one place

Reply:
Good point! 👍 Corrected.